src/main/native/build_windows_jni.sh

```diff
@@ -51,10 +51,19 @@ if [ -z "${BAZEL_VS+set}" ]; then
 fi
 VSVARS="${BAZEL_VS}/VC/VCVARSALL.BAT"
 
-# Check if Visual Studio 2017 is installed. Look for it at the default location.
+# Check if Visual Studio 2017 is installed. Look for it at the default
+# locations.
 if [ ! -f "${VSVARS}" ]; then
-  VSVARS="C:/Program Files (x86)/Microsoft Visual Studio/2017/BuildTools/VC/"
-  VSVARS+="Auxiliary/Build/VCVARSALL.BAT"
+  VSVARS="C:/Program Files (x86)/Microsoft Visual Studio/2017/"
+  VSEDITION="BuildTools"
+  if [ -d "${VSVARS}Enterprise" ]; then
```

Comment:
IMO the bit cleaner way would be to search for `VC/Auxiliary/Build/VCVARSALL.BAT` and from that infer the possible editions.